Cargo.toml

```diff
@@ -9,4 +9,4 @@ repository = "https://github.com/dusk-network/nstack"
 keywords = ["merkle", "datastructure", "stack"]
 
 [dependencies]
-kelvin = "0.18"
\ No newline at end of file
+kelvin = "0.19.0"
```

Comment:
nit: add the new line at the end of the file.

Reply:
Done.